cliutils/port_forward.go

```diff
@@ -0,0 +1,115 @@
+package cliutils
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	errors "github.com/rotisserie/eris"
+)
+
+// Call kubectl port-forward. Callers are expected to clean up the returned portFwd *exec.cmd after the port-forward is no longer needed.
+func PortForward(namespace string, resource string, localPort string, kubePort string, verbose bool) (*exec.Cmd, error) {
+
+	/** port-forward command **/
+
+	portFwd := exec.Command("kubectl", "port-forward", "-n", namespace,
+		resource, fmt.Sprintf("%s:%s", localPort, kubePort))
+
+	portFwd.Stderr = os.Stderr
+	if verbose {
+		portFwd.Stdout = os.Stdout
+	}
+
+	if err := portFwd.Start(); err != nil {
+		return nil, err
+	}
+
+	return portFwd, nil
+
+}
+
+// Call kubectl port-forward and make a GET request.
+// Callers are expected to clean up the returned portFwd *exec.cmd after the port-forward is no longer needed.
+func PortForwardGet(ctx context.Context, namespace string, resource string, localPort string, kubePort string, verbose bool, getPath string) (string, *exec.Cmd, error) {
+
+	/** port-forward command **/
+
+	portFwd, err := PortForward(namespace, resource, localPort, kubePort, verbose)
+	if err != nil {
+		return "", nil, err
+	}
+
+	localCtx, cancel := context.WithTimeout(ctx, time.Second*30)
```

Comment:
This change from 3 sec to 30 sec is intentional, right?

Reply:
Nope!

Reply:
Oh sorry. I think I was looking at the PR, and someone probably changed it since.